integration-cli/docker_cli_start_test.go

```diff
@@ -75,32 +75,26 @@ func (s *DockerSuite) TestStartRecordError(c *check.C) {
 	// when container runs successfully, we should not have state.Error
 	dockerCmd(c, "run", "-d", "-p", "9999:9999", "--name", "test", "busybox", "top")
 	stateErr, err := inspectField("test", "State.Error")
-	c.Assert(err, check.IsNil)
-	if stateErr != "" {
-		c.Fatalf("Expected to not have state error but got state.Error(%q)", stateErr)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("stateErr: %s", stateErr))
+	// Expected to not have state error
+	c.Assert(stateErr, checker.Equals, "")
 
 	// Expect this to fail and records error because of ports conflict
 	out, _, err := dockerCmdWithError("run", "-d", "--name", "test2", "-p", "9999:9999", "busybox", "top")
-	if err == nil {
-		c.Fatalf("Expected error but got none, output %q", out)
-	}
+	// Expected error but got none
+	c.Assert(err, checker.NotNil, check.Commentf("out: %s", out))
```

Comment:
As we have discussed in #16854, comment above c.Assert statement will be printed out together with check.Commentf, do you still see a need to put comment in check.Commentf?
@vdemeester